src/main/java/me/gallowsdove/foxymachines/implementation/weapons/Elucidator.java

```diff
@@ -0,0 +1,48 @@
+package me.gallowsdove.foxymachines.implementation.weapons;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import me.gallowsdove.foxymachines.Items;
+import me.gallowsdove.foxymachines.utils.Utils;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Elucidator extends OnHitWeapon {
+    public Elucidator(boolean customMobs) {
+        super(Items.WEAPONS_AND_ARMORS_ITEM_GROUP, Items.ELUCIDATOR, RecipeType.ANCIENT_ALTAR, new ItemStack[]{
+                Items.EQUANIMOUS_GEM, Items.CURSED_SWORD, Items.EQUANIMOUS_GEM,
+                Items.BUCKET_OF_BLOOD, customMobs ? Items.PIXIE_QUEEN_HEART : Items.DEMONIC_PLATE, Items.BUCKET_OF_BLOOD,
+                Items.EQUANIMOUS_GEM, Items.CELESTIAL_SWORD, Items.EQUANIMOUS_GEM
+        });
+    }
+
+    @Override
+    @ParametersAreNonnullByDefault
+    public void onHit(ThreadLocalRandom random, EntityDamageByEntityEvent event, HumanEntity humanoid, LivingEntity entity) {
+        // Damage III
+        event.setDamage(event.getDamage() * 2);
+
+        // Armor Ignorance I
+        Utils.dealDamageBypassingArmor(entity, (event.getDamage() - event.getFinalDamage()) * 0.05);
```

Comment:
I am not quite sure about this, since 0.05 is quite a big change from 0.064, tho it might not seem like it. I've chosen these values after quite a lot of testing, so I'm not sure whether this is a justifiable nerf. (I do get you've done it for consistency, but I'm thinking whether it wouldn't be better to just keep the old value, not like anyone else is gonna notice)

Reply:
I can change it back, I totally get it! I'll set it to be II of the enchantment and switch the current III to V to make the scaling seem more accurate